.github/workflows/link.yml

```diff
@@ -17,6 +17,12 @@ jobs:
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
+      - uses: actions/checkout@v1
+      - name: Run Tests
+        run: |
+          npm install
+          npm test
+          
```

Comment:
Does this work? As per our Discord conversation, I thought this didn't work. If not, remove it from the PR to get my approval.

Reply:
Update : as per our Discord conversation, I can have a look at whether or not this works.